omero/developers/Server/ObjectGraphs.txt

```diff
@@ -8,9 +8,10 @@ Model graph operations
   deleting users' rendering settings for that image, also the links to
   any datasets that the image is in. Version 5.1.0 of the OMERO server
   includes a new algorithm for determining which model objects to act
-  on. Understanding the details of the new implementation substantially
-  assists in debugging or creating server operations that act on the
-  directed graph of model objects.
+  on. Clients should be :doc:`GraphsMigration` because the previous ones
```

Comment:
I'd include "using the new 5.1..." text here. 1) the capitalization is off and 2) the text could eventually change and make this hard to read.